### PR TITLE
Implement picker styling

### PR DESCRIPTION
**Pull request by RMIAOOOOO**
Implement styling props for `<Picker>`

`<Picker>`'s toolbar and items can now be styled like 
```
        <Picker
          visible={visible}
          value={selectedValue}
          items={ITEMS}
          onDismiss={this.onDismiss}
          onDone={this.onDone}
          onCancel={this.onCancel}
          toolbarStyle={{ backgroundColor: "rgb(72,72,74)" }}
          itemsStyle={{
            backgroundColor: "rgb(58,58,60)",
            color: "rgb(242,242,247)",
          }}
        />
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,31 @@ git commit
...
+static BOOL RCTParseUnused(const char **input)
+{
+  return RCTReadString(input, "__unused") ||
+         RCTReadString(input, "__attribute__((__unused__))") ||
+         RCTReadString(input, "__attribute__((unused))");
+}
+```
+
+then the example project can be compiled.
+
```
Comment (cychiuae) [inline README.md]: @louischan-oursky 

I also encountered problem when I tried to run example. I think we can try to update RN version of example to see problem still exists or not.

Comment (louischan-oursky) [reply README.md]: The last two commits look good to me. Maybe we can drop the first commit and keep the last two? Let me open a new issue to deal with the compilation problem of the example.

Comment (cychiuae) [reply README.md]: Agree

**Comment by cychiuae** [comment]
Merge via 56cdbeb

**Comment by louischan-oursky** [comment]
Thank both you!

**Pull request closed without merging**